exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/JsonSerializer.java

```diff
@@ -71,6 +71,11 @@ protected void writeFixed64Value(long value) throws IOException {
     generator.writeString(Long.toString(value));
   }
 
+  @Override
+  protected void writeFixed32(ProtoFieldInfo field, int value) throws IOException {
+    generator.writeStringField(field.getJsonName(), Integer.toString(value));
```

Comment:
`writeNumberField` for this one, 32-bit is number while 64-bit is string

https://developers.google.com/protocol-buffers/docs/proto3#json